### ISSUE TITLE
support for "skipsteps = ['permission']"

### ISSUE THREAD

**Issue by jhgoebbert** [easybuilders/easybuild#661]
We need to set special permissions for the files/folders of single modules.  
So I would like to be able to do the following in an eb-file.
```
skipsteps = ['permissions']
postinstallcmds = ['chmod -R ugo-w %(installdir)s/config', 
                   'chmod -R ugo-w %(installdir)s/share',
]
```
It is essential that also the owner of the files/folders has no write permission.

Currently easybuild does not allow to skip the `permissions` step.

**Comment by akesandgren** [comment]
eb ... --umask=0222

**Comment by jhgoebbert** [comment]
Thank you @akesandgren 

but this way I can not store this important installation step in the eb-file.  
Hence it will for sure get lost and forgotten in the hundreds of other packages when one installs a whole new stage on another machine or do an update.
(second, this option would change the permissions of all files/directories one might not want)

I do not think this is a solution for production.

**Comment by akesandgren** [comment]
Yeah, there are plans to make umask an easyconfig param, but noone has gotten around to fixing the last bits that are missing.

So I usually add a comment to the easyconfig that it has done use --umask=xxx

**Comment by jhgoebbert** [comment]
Yes, a comment should do it.

Thanks for all your hard work on easybuild. I love it and it makes life soooooo much easier!

**Comment by jhgoebbert** [comment]
> Yeah, there are plans to make umask an easyconfig param, but noone has gotten around to fixing the last bits that are missing.

Being able to skip the `permissions` step would already do (and would be more flexible in combination with `postinstallcmds`.
